src/gtkspellcheck/spellcheck.py

```diff
@@ -495,121 +477,181 @@ def check_range(self, start, end, force_all=False):
                 break
             word_start = word_end.copy()
 
-    def _languages_menu(self):
-        def _set_language(item, code):
-            self.language = code
+    def _setup_actions(self) -> None:
```

Comment:
This is just for GTK 4, isn't it? Maybe we can prefix it with `_gtk4`? Just to make clear that this method is not expected to do anything meaningful for GTK 3. That would generally be my approach. If something is just for one of the versions, give it a prefix.

Reply:
Done. Originally I had some of these throughout.